src/utils/composeUtil.test.ts

```diff
@@ -0,0 +1,60 @@
+import { composeUtil } from "./composeUtil";
+import { testConfig } from "./testUtils";
+import { createCss, createTokens, IAtom } from "@stitches/core";
```

Comment:
I would prefer to import external dependencies at the top :)

Reply:
sure, do you want to add eslint to enforce this kind of rules? I didn't want to set it up as part of this work but you can have stuff like https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/first.md to save time in future PRs.

I've done the change in [`a0e5dc1` (#2)](https://github.com/hauptrolle/stitches-utils/pull/2/commits/a0e5dc12f39096b6e43abf16cd6a2d0cdbc86c0c)

Reply:
Perfect! :) Yes, I created an [issue](https://github.com/hauptrolle/stitches-utils/issues/4) to keep track of adding eslint.